packages/header-checker-lint/src/header-checker-lint.ts

```diff
@@ -87,19 +105,11 @@ export = (app: Application) => {
     try {
       filesResponse = await context.github.pulls.listFiles(listFilesParams);
     } catch (err) {
-      app.log.error('---------------------');
-      app.log.error(err);
+      logger.error('---------------------');
```

Comment:
I don't sully understand how to add custom labels in the logs, I know @azizsonawalla was showing this functionality off today. Would this be an option to help this log jump out, rather than `-------`.

Reply:
I will remove the '-----' logging - it's not useful